### DISCUSSION TITLE
miscellany/coding/20241122-meson-for-cpp

### DISCUSSION THREAD

**Discussion by giscus** [kands-code/kands-code.github.io#84] [General]
# miscellany/coding/20241122-meson-for-cpp

找到属于自己的光

https://blogs.qarks.top/miscellany/coding/20241122-meson-for-cpp.html

**Comment by kands-code** [comment]
如果要指定编译器，可以通过 `CC` 等环境变量

**Comment by kands-code** [comment]
如果想要一个较为全面的模板，或许可以使用这个仓库作为模板：[ks-meson-template](https://github.com/kands-code/ks-meson-template)
